app/models/metrics_updates.rb

```diff
@@ -0,0 +1,9 @@
+class MetricsUpdates
```

Comment:
Can you think of a better name that `MetricsUpdate`? Maybe `UpdateLog`, since the job of this is to log data about the update cycles? Or something else, but I think `MetricsUpdate` doesn't clearly communicate what this is for.